gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -325,6 +354,29 @@ public void delete_throwsExceptionWhenHadoopPathNull() throws IOException {
     assertThat(exception).hasMessageThat().startsWith("hadoopPath must not be null");
   }
 
+  @Test
+  public void open_IOstatistics() throws IOException {
+    GoogleHadoopFileSystem myGhfs = createInMemoryGoogleHadoopFileSystem();
+    Path testRoot = new Path("/directory1/");
+    myGhfs.mkdirs(testRoot);
+    FSDataOutputStream fout = myGhfs.create(new Path("/directory1/file1"));
+    fout.writeBytes("data");
+    fout.close();
+    myGhfs.open(new Path("/directory1/file1"));
+    assertThat(myGhfs.getIOStatistics().counters().get("op_open")).isEqualTo(1);
```

Comment:
Can we use enums instead of string literals for metrics here ?

Reply:
Updated the code with enums